lib/launchApplication.js

```diff
@@ -29,12 +32,29 @@ module.exports = async function launchApplication(isDebug) {
         });
     }
 
-    let results = await controller.start();
+    if (wgtFilePath) {
```

Comment:
I think input package id directly every time when launch with wgt file is not very usability for user.
how about get package id from config.xml in wgt file.
I mean, you unzip wgt file into temp folder to get confilg.xml file and delete it.

Reply:
@prate3k Refer to followed package.
https://stackoverflow.com/questions/39705209/node-js-read-a-file-in-a-zip-without-unzipping-it

Reply:
Thanks for the suggestion :) Checking this.

Reply:
Found out a way to use existing `decompress` and `xml2js` module for extracting application id. I've pushed the changes. Can you please re-review.